server/xpub-model/entities/manuscript/index.test.js

```diff
@@ -225,15 +330,54 @@ describe('Manuscript', () => {
           status: 'INITIAL',
         }).save(),
       ).rejects.toThrow())
+
+    it('does not overwrite an updated manuscript', async () => {
+      await testOverwritingWithOldData(userId)
+    })
   })
 
   describe('all()', () => {
     it("returns users's manuscripts only", async () => {
       const secondUserId = uuid()
-      await new Manuscript({ createdBy: userId }).save()
-      await new Manuscript({ createdBy: secondUserId }).save()
+      await new Manuscript({
+        createdBy: userId,
+      }).save()
+      await new Manuscript({
+        createdBy: secondUserId,
+      }).save()
       const loadedManuscripts = await Manuscript.all(userId)
       expect(loadedManuscripts).toHaveLength(1)
     })
   })
 })
+
+const testOverwritingWithOldData = async (userId, refresh = false) => {
```

Comment:
New function that test the overwriting with old data should throw if the object is not refreshed or work if it is refreshed